README.md

```diff
@@ -119,4 +120,5 @@ If you plan to contribute to the project, please refer to [tests overview articl
 ## Other useful links
 
 * [FAQ & Troubleshooting](docs/faq.md)
-* [Version history](docs/changelog.md)
\ No newline at end of file
+* [Version history](docs/changelog.md)
+* [Steps to Intregrate New Service](docs/service_integration.md)
```

Comment:
One important thing here is to point out that the info is relevant for contributors only, not end-users.

I guess, it better fits into "Contributing" section of the documentation (instead of general-purpose "Other useful links" section). In "contributing" we can say - if you want to integrate ..., then - follow this guide.